src/interfaces/ICoWSwapEthFlow.sol

```diff
@@ -51,4 +51,20 @@ interface ICoWSwapEthFlow {
         external
         view
         returns (bytes4 magicValue);
+
+    /// @dev This function reads the  chain's native token balance of this contract (e.g., ETH for mainnet) and converts
+    // the entire amount to its wrapped version (e.g., WETH).
+    function wrapAll() external;
+
+    /// @dev This function takes the specified amount of the chain's native token (e.g., ETH for mainnet) stored by this
+    /// contract and converts it to its wrapped version (e.g., WETH).
+    ///
+    /// @param amount The amount of native tokens to convert to wrapped native tokens.
+    function wrap(uint256 amount) external;
```

Comment:
Will we keep both around here in the contract or choose one and delete the other?

Update (@nlordell was faster) - I like the suggestion to keep both.